{{ cookiecutter.git_repo_name }}/setup.cfg

```diff
@@ -0,0 +1,8 @@
+{%- if cookiecutter.support_python_2 == 'y' -%}
+[bdist_wheel]
+universal=1
+
+{% endif -%}
+
+[tool:pytest]
+addopts =-x -v -rw --durations=10 --cov={{ cookiecutter.python_package_name }} --cov-report=term-missing
```

Comment:
Let's remove

    --cov={{ cookiecutter.python_package_name }} --cov-report=term-missing

to simplify this line.